src/components/Slider/SliderItem/SliderItem.tsx

```diff
@@ -11,15 +14,17 @@ type Props = {
 }
 
 export const SliderItem: FC<Props> = ({ picture }) => {
+    const { t, i18n } = useTranslation()
+
     return (
-        <div className={styles.wrapper}>
+        <div className={cn(styles.wrapper, i18n.language === 'ua' && styles.wrapperUa)}>
```

Comment:
'ua' - magic variable,  move it to `enum` and use it

Reply:
added to constants